subsys/net/lib/sockets/sockets_tls.c

```diff
@@ -189,6 +189,22 @@ static struct tls_context *tls_alloc(void)
 	return tls;
 }
 
+/* Allocate new TLS context and copy the content from the source context. */
+static struct tls_context *tls_clone(struct tls_context *source_tls)
+{
+	struct tls_context *target_tls;
+
+	target_tls = tls_alloc();
+
```

Comment:
Remove empty line, we normally do not have empty line between variable setting and if() statement that checks it.